File: SustitucionMOAModel/Dto/OrdenDeCarga/CrearOrdenEnSAPRequest.cs
namespace SustitucionMOAModel.Dto.OrdenDeCarga
{
	public class CrearOrdenEnSAPRequest
	{
		public int IdOrdenDeCarga { get; set; }
		public string ClienteCodigo { get; set; }
		public string ContratoSAP { get; set; }
		public string CorredorCodigo { get; set; }
		public int Cantidad { get; set; }
		public string MaterialCodigoSAP { get; set; }
		public string NumeroPedidoIngresado { get; set; }
		//public string ValidarKg { get; set; }
		public string MailUsuarioSAP { get; set; }
	}
}

File: SustitucionMOA/src/app/common/models/ordenes-de-carga/crearOrdenEnSAPRequest.ts
export interface CrearOrdenEnSAPRequest {
    IdOrdenDeCarga: number;
    ClienteCodigo: string;
    ContratoSAP: string;
    CorredorCodigo: string;
    Cantidad: number;
    MaterialCodigoSAP: string;
    NumeroPedidoIngresado: string;
    // ValidarKg: string;
    MailUsuarioSAP: string;
}

export interface CrearOrdenEnSAPResponse {
    ResultCreation: boolean;
    Error: string;
}